lib/ssl/src/ssl_pem_cache.erl

```diff
@@ -136,6 +136,7 @@ init([Name]) ->
     Interval = pem_check_interval(),
     erlang:send_after(Interval, self(), clear_pem_cache),
     erlang:system_time(second),
+    proc_lib:set_label({tls, pem_cache, Name}),
```

Comment:
will it not be used for DTLS? does it make a sense to have protocol here?

Reply:
Actually this was a registered process so I removed the label as it is not needed!